tests/src/test/resources/META-INF/dirigible/camel_test/calledFromCamel.js

```diff
@@ -0,0 +1,7 @@
+exports.onMessage = (message) => {
+    let messageBody = message.getBodyAsString();
+    let modifiedMessageBody = messageBody + " -> calledFromCamel.js handled this message";
```

Comment:
Use const and string interpolation

Reply:
changed to string interpolation and const